### DISCUSSION TITLE
IC setting for a trapezium in the phase field model

### DISCUSSION THREAD

**Discussion by emmelines** [idaholab/moose#19336] [Q&A Modules: Phase field] [answered]
Hi,

I am building up a new simulation. 
I need a trapezium in a box as the initial condition in my model.
Is there any suggestion to build this? (any recommend build-up IC app?)

Thank you.

**Comment by dschwen** [comment, marked as answer]
Check out [this answer](https://github.com/idaholab/moose/discussions/19333#discussioncomment-1615636).

**Reply by emmelines** [reply]
Hi,

Thank you for your answer. I tried to put it together. 
Do you have any ideas on how should I fix the vtkPNGReader problem shown below:

Reading image(s)...
ERROR: In ../IO/Image/vtkPNGReader.cxx, line 44
vtkPNGReader (0x7fbd90af37b0): Unable to open file /Users/emmelineshue/projects/moose/modules/phase_field/phase_field/test_003.png

          ...image read finished

Thank you so much.

**Reply by GiudGiud** [reply]
what is this directory?
phase_field/phase_field/  

I dont have it
